gradle.properties

```diff
@@ -6,3 +6,11 @@ mockitoVersion=1.9.5
 junitVersion=4.11
 log4jVersion=1.2.17
 jodaTimeVersion=2.3
+commonsCodecVersion=1.9
+commonsLoggingVersion=1.2
+commonsHTTPAsyncClientVersion=4.1
+commonsHTTPAsyncClientCacheVersion=4.1
+commonsHTTPClientVersion=4.4.1
+commonsHTTPClientCacheVersion=4.4.1
+commonsHTTPCoreVersion=4.4.1
+commonsHTTPCoreNIOVersion=4.4.1
```

Comment:
Minor, of course, but that would be better to configure the IDE to add an new line in the end of file.
At least, to avoid such a comments in the future :smile: